basket/petition/views.py

```diff
@@ -0,0 +1,35 @@
+import uuid
+
+from django.conf import settings
+from django.http import JsonResponse
+from django.shortcuts import redirect
+
+from basket.petition.forms import PetitionForm
+
+
+# Doing this here to not affect the rest of basket.
+def _add_cors(response):
+    response["Access-Control-Allow-Origin"] = "https://open.mozilla.org"
```

Comment:
Q: do we want this to be configurable? Probably not for now

Reply:
Good catch. I can move it to settings. I already have one there so it makes sense for this too.